src/components/security-analysis-result.js

```diff
@@ -134,19 +154,19 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
                 postContingencyResult.status !== 'CONVERGED'
             ) {
                 rows.push({
-                    contingencyIndex: index,
+                    //contingencyIndex: index,
                     contingencyId: postContingencyResult.contingency.id,
                     computationStatus: postContingencyResult.status,
                     violationCount:
                         postContingencyResult.limitViolationsResult
                             .limitViolations.length,
-                    _group: index,
-                    _root: true,
+                    //  _group: index,
+                    //  _root: true,
                 });
```

Comment:
same here and all the commented field below

Reply:
Done